migrations/sql/V2021.07.09_1359__WorkflowRunReports.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE orderlyweb_workflow_run_reports
+(
+    id              INTEGER PRIMARY KEY,
+    workflow_key    TEXT      NOT NULL,
+    key             TEXT      NOT NULL UNIQUE,
+    name            TEXT      NOT NULL,
```

Comment:
This is still inconsistent with the `orderlyweb_report_run` table, where the column is named `report`. I'm not saying `name` is less appropriate than `report`, but ultimately consistency is preferable even if we don't end up unifying these tables.